subnet/deployment-generator/docker/Dockerfile

```diff
@@ -31,5 +38,4 @@ RUN chmod +x /app
 
```

Comment:
why run chmod +x /app, the folder should already be executable?

Reply:
there was issue with start script I think. But it's also catch-all in case any new files are added in the future.

Reply:
still not good. Better do proper permission control, you can do 
```chmod -R +x /app/*.sh```
But I would suggest specify which file needs to change permission. Not all of them